e2e/features/appeal-validation.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Other Appeal Numbers are validated
+
+  Background:
+    Given I am signed in as a Legal Rep
+    And I create a new case
+    And I complete the screening questions page
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+    And I complete the Your client's address page
+    And I complete the Why is your client appealing? page
+    And I complete the New matters page
+
+  @create-case @alternate @RIA-635
+  Scenario Outline: Invalid appeal reference
+
+    Given I am on the Has your client appealed against any other UK immigration decisions? page
+    And I select Yes for the Other appeals field
+    And I click the Continue button
+    And I am on the Has your client appealed against any other UK immigration decisions? page
+    And I click the Add new button
```

Comment:
Was there a reason you didn't use:
`And I add a new item to the Appeal number collection`
rather than clicking a button?

Not a problem for this form now, but clicking the button directly won't work if more collections are added to this page.